src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -1,9 +1,12 @@
 package mate.academy.hibernate.relations.dao.impl;
 
 import java.util.Optional;
+import mate.academy.hibernate.relations.DataProcessingExeption;
```

Comment:
There is a typo in the imported exception class `DataProcessingExeption`. It should be `DataProcessingException` to match the corrected class name.